_data/toc.yaml

```diff
@@ -1794,6 +1800,17 @@ manuals:
   - path: /docker-hub/release-notes/
     title: Release notes
 
+- sectiontitle: Docker Scout
```

Comment:
Should this be `Docker Scount (EA/Early Access)` to keep in line with how we do beta features - `Wasm (Beta) ?

Reply:
Yes, that's a good catch!